CITATIONS.bib

```diff
@@ -0,0 +1,14 @@
+@misc{tanaka2023xiplot,
```

Comment:
The citation button is also enabled for bib files (see e.g. https://github.com/edahelsinki/slisemap). The downside of using a bib file is that it only exports as a bib file. The upside is that bib files can contain multiple citations (for when we get the non-arXiv reference).

Reply:
Oh I did not know about that, that's quite cool! In that case, we can go with the bib file (since there a numerous converters from bib to other formats online).

For completeness sake, the citation file format also kind of supports multiple citations, though only as multiple identifiers and a preferred reference, e.g. https://github.com/juntyr/necsim-rust/blob/main/CITATION.cff